modules/common/src/main/scala/surge/internal/tracing/TracingHelper.scala

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2017-2021 UKG Inc. <https://www.ukg.com>
+
+package surge.internal.tracing
+
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.api.trace.{ Span, SpanBuilder, Tracer }
+
+import scala.jdk.CollectionConverters._
+
+private[surge] trait SpanSupport {
+  protected def tracer: Tracer
+
+  def newSpan(operationName: String): Span = {
+    tracer.spanBuilder(operationName).setNoParent().startSpan()
+  }
+
+  def createSpan(operationName: String): Span = newSpan(operationName)
+
+  def childSpan(operationName: String, parentSpan: Span): Span = {
+    tracer.spanBuilder(operationName).setParent(io.opentelemetry.context.Context.root().`with`(parentSpan)).startSpan()
+  }
+}
+
+trait TracingHelper {
```

Comment:
this contains some "aliases" for easy migration from OpenTracing to OpenTelemetry